src/Components/WebAssembly/Server/src/Microsoft.AspNetCore.Components.WebAssembly.Server.csproj

```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Reference Include="Newtonsoft.Json" Version="13.0.2" />
```

Comment:
Could we avoid adding this dependency? Or if there's some reason why `System.Text.Json` is insufficient, could you clarify what it is?

Reply:
This is the same used on BrowserDebugProxy, if you prefer I can try to use System.Text.Json.

Reply:
Ah, I see, so it won't change the dependencies as far as customers are concerned.

However, having the direct dependency here does mean more dependabot-type noise and TBH it's just really surprising to see Json.NET code show up when we've consistently used STJ elsewhere in this repo. If it's not too disruptive to switch over to STJ for this that would be helpful! If that's impractical or a lot of work let me know.

Reply:
Removed :)